### ISSUE TITLE
Re-send approval email for users affected by #246 

### ISSUE THREAD

**Issue by jnordberg** [steemit/faucet#259]
We need a script that queries the database for users who have been approved but not completed signup and re-sends the email with a new token that does not expire.

Refs #246

**Comment by pkattera** [comment]
There may be up to 126K users who have been approved more than 7 days ago, who will no longer be able to finish creating an account, so we need to send them all an email reminder with a new link with a longer expiry time.

**Comment by agerhbour** [comment]
Yes please. I think I have royally stuffed up my account by trying to sign up again. I got several emails asking to confirm my email address, they all take me to a page that say thankyou for confirming your email, but then nothing happens.

**Comment by urspine** [comment]
I am getting 'Invalid token error' when I try to complete the registration. Plz help.

**Comment by hLxiao9** [comment]
With the same problem. Plz help.

**Comment by armaccloud** [comment]
Hi, I am having this issue too. Well within the 7 days for verification.
Username: arnomuhren

**Comment by nelsnelson** [comment]
I am having this issue also.

**Comment by gl2748** [comment]
hey @nelsnelson how long after your first attempt to signup did you wait before clicking on the email confirmation link...

**Comment by urspine** [comment]
Still no help from steemit. I hope you guys will solve it fast. thanks.

**Comment by lgendrot** [comment]
The fix for this has been merged in #269, so the admins should be able to re-send the emails now. Is there any reason it hasn't happened yet?

**Comment by oilsinwater** [comment]
yes wondering what the process to get a new email to setup new account. been waiting for a couple of months now...

**Comment by gl2748** [comment]
Need to address #275 before continuing with rescue emails.

**Comment by jnordberg** [comment]
Emails where never sent out, reopening to track completion. Refs #296

**Comment by gl2748** [comment]
These emails were all sent...

**Comment by nelsnelson** [comment]
@gl2748 These e-mails were supposed to be the re-sending of the "One last step" e-mails to activate an account using a token link, right?

I did not receive any e-mails from steemit whatsoever since that original activation e-mail.

```
Hi again,

Good news! Your account information has been verfied.

Click on this link to finish setting up your account.

https://signup.steemit.com/create-account?token=[elided]

Cheers,
Your friends at Steemit
```

To be clear, the `https://signup.steemit.com/create-account?token=...` indicated that I would receive an e-mail confirming my account activation within 7 days.  I never received any further correspondence from steemit.

**Comment by gl2748** [comment]
@nelsnelson apologies. There are a few different cases, and groups that need an email, and I was mixed up. We will get to you asap.

**Comment by lgendrot** [comment]
@gl2748 Just going to note that @nelsnelson is not the only one in that same position. I received the same email, yet the token was expired by the time I got to it. I never received a re-sent email.

**Comment by gl2748** [comment]
Superseded by https://github.com/steemit/steempunks/issues/299

**Comment by pkattera** [comment]
To confirm we can still do this with legal team

**Comment by TimCliff** [comment]
https://www.reddit.com/r/steemit/comments/8opw7i/finish_setting_up_a_steemit_account/

**Comment by Ratsound** [comment]
Guys I have the same problem, please resend the email. My username @ratsound

**Comment by krischik** [comment]
More then one of my friend who tried to open an account got stuck. The account opening process really need some “retry” false saves.
